server/api/users.js

```diff
@@ -1,19 +1,95 @@
 const router = require('express').Router();
-const { User } = require('../db');
+const { User, Order, OrderItem } = require('../db');
 
+// get all users
 router.get('/', (req, res, next) => {
   User.findAll()
     .then(users => res.send(users))
     .catch(next);
 });
 
-// TODO - for cart/order functionality:
+// get all orders for specific user
+router.get('/:id/orders', (req, res, next) => {
```

Comment:
Let's be consistent in this file, so that we use 
userId
orderId
orderItemId
and do not user id
(just pointing out the first instance, need to change all of the others as well)
```suggestion
router.get('/:userId/orders', (req, res, next) => {
```